docs/project/contributing.md

```diff
@@ -26,8 +30,32 @@ To help with this task we have implemented the commitizen to all libraries, just
 - Push to your fork and submit a pull request
```

Comment:
I think we could to slit it into sessions
## how to "find" a issue
- explain what is a issue
- where is the issues
- < gif >
### how to open a issue
- < gif > 

## "downloading" and running the code
- how to fork, why fork...
- create branch
- run tests
< gif >

## commting and poushing
- how to use commitizen
- < gif >

# PR
- what is a PR
- how to open a great PR
- < gif >


##